src/bot/api.rs

```diff
@@ -228,16 +236,16 @@ impl Bot {
         )
     }
 
-    pub fn send_poll<C, Q, O>(
+    pub fn send_poll<'a, C, Q, O>(
         &self,
         chat_id: C,
         question: Q,
         options: O,
     ) -> SendPoll
     where
-        C: Into<ChatId>,
-        Q: Into<String>,
-        O: Into<Vec<String>>
+        C: Into<ChatId<'a>>,
+        Q: Into<Cow<'a, str>>,
+        O: Into<Cow<'a, [str]>>,
```

Comment:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=5047068c9c7876032b390f87062994d2

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=a8cd77287a03e1752b9ea262a006641c

Is it OK?

Reply:
Nope

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f0bde0391102c726f396fd20c41b809b

Reply:
But `working(vec!["Oops"]);` won't work.